src/test/java/com/amazon/opendistroforelasticsearch/security/IndexIntegrationTests.java

```diff
@@ -473,7 +474,13 @@ public void testCCSIndexResolve() throws Exception {
 
         res = rh.executeGetRequest("/*:.abc-6,.abc-6/_search", encodeBasicHeader("ccsresolv", "nagilum"));
         Assert.assertEquals(HttpStatus.SC_OK, res.getStatusCode());
+
         //TODO: Change for 25.0 to be forbidden (possible bug in ES regarding ccs wildcard)
+
+        // invalid_index_name_exception should be thrown and responded when invalid index name is mentioned in requests.
+        res = rh.executeGetRequest(URLEncoder.encode("_##pdt_data/_search", "UTF-8"), encodeBasicHeader("ccsresolv", "nagilum"));
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, res.getStatusCode());
+        Assert.assertTrue(res.getBody().contains("invalid_index_name_exception"));
```

Comment:
Is it possible to create separate test as it is not related to cross cluster search?

Reply:
Done